Signal/src/ViewControllers/ConversationView/Cells/OWSQuotedMessageView.m

```diff
@@ -147,114 +220,47 @@ - (void)createContents
-        } else {
-            [quotedAuthorLabel autoPinTrailingToSuperviewMarginWithInset:self.quotedContentHInset];
-        }
-        [quotedAuthorLabel autoSetDimension:ALDimensionHeight toSize:self.quotedAuthorHeight];
-        [quotedAuthorLabel setContentHuggingLow];
-        [quotedAuthorLabel setCompressionResistanceLow];
-    }
-
-    {
-        // Stripe and text container.
-        UIView *stripeAndTextContainer = [UIView containerView];
-        [self addSubview:stripeAndTextContainer];
-        [stripeAndTextContainer autoPinEdge:ALEdgeTop
-                                     toEdge:ALEdgeBottom
-                                     ofView:quotedAuthorLabel
-                                 withOffset:self.quotedAuthorBottomSpacing];
-        [stripeAndTextContainer autoPinLeadingToSuperviewMarginWithInset:self.quotedContentHInset];
-        if (quotedAttachmentView) {
-            [stripeAndTextContainer autoPinTrailingToLeadingEdgeOfView:quotedAttachmentView
-                                                                offset:self.quotedAttachmentHSpacing];
-        } else {
-            [stripeAndTextContainer autoPinTrailingToSuperviewMarginWithInset:self.quotedContentHInset];
-        }
-        [stripeAndTextContainer autoPinBottomToSuperviewMarginWithInset:self.quotedTextBottomInset];
-        [stripeAndTextContainer setContentHuggingLow];
-        [stripeAndTextContainer setCompressionResistanceLow];
-
-        // Stripe.
-        UIView *quoteStripView = [UIView containerView];
-        quoteStripView.backgroundColor = self.highlightColor;
-        quoteStripView.userInteractionEnabled = NO;
-        quoteStripView.layer.cornerRadius = self.quotedReplyStripeRounding;
-        [stripeAndTextContainer addSubview:quoteStripView];
-        [quoteStripView autoPinHeightToSuperview];
-        [quoteStripView autoPinLeadingToSuperviewMargin];
-        [quoteStripView autoSetDimension:ALDimensionWidth toSize:self.quotedReplyStripeThickness];
-        [quoteStripView setContentHuggingVerticalLow];
-        [quoteStripView setContentHuggingHorizontalHigh];
-        [quoteStripView setCompressionResistanceHigh];
-
-        // Text.
-        UILabel *quotedTextLabel = [self configureQuotedTextLabel];
-        [stripeAndTextContainer addSubview:quotedTextLabel];
-        [quotedTextLabel autoPinTopToSuperviewMarginWithInset:self.quotedReplyStripeVExtension];
-        [quotedTextLabel autoPinBottomToSuperviewMarginWithInset:self.quotedReplyStripeVExtension];
-        [quotedTextLabel autoPinLeadingToTrailingEdgeOfView:quoteStripView offset:self.quotedReplyStripeHSpacing];
-        [quotedTextLabel autoPinTrailingToSuperviewMargin];
-        [quotedTextLabel setContentHuggingLow];
-        [quotedTextLabel setCompressionResistanceLow];
+        [hStackView addArrangedSubview:quotedAttachmentView];
+    } else {
+        UIView *emptyView = [UIView containerView];
```

Comment:
Why do we need a zero space empty view in the stack?

Reply:
I've added a comment.